how-to-release.md

```diff
@@ -0,0 +1,274 @@
+Each release should be executed by a Release Manager, who is selected/proposed by the steering committee. This document describes the process that the Release Manager follows to perform a release. Any changes to this process should be discussed and adopted on the dev@ mailing list.
+
+This release process should be strictly followed by Pravega and also followed by all the ecosystem projects inside Pravega organization by default, but Pravega ecosystem projects can have its own release instruction page that points to this one as a reference and then explains what is different.
+
+If you are releasing the project, then you must read and follow the instructions carefully. The releases are a public face of the project and most users interact with the project only through the releases, so they need to be treated with great importance.
+
+# Overview
+
+![release process](https://cwiki.apache.org/confluence/download/attachments/73631092/release-guide-1.png)
```

Comment:
this is a reference to a external website, please remove (it is on the ASF confluence wiki)

Reply:
Fixed.